zproject/prod_settings_template.py

```diff
@@ -837,3 +837,15 @@
 
 ## Directory containing Markdown files for the server's policies.
 # POLICIES_DIRECTORY = "/etc/zulip/policies/"
+
+# Schedule deactivated realm data deletion
+
+# Minimum number of days to be given in realm data
+# deletion schedule when deactivating a realm.
+# Give 0 if there is no minimum value.
+# MIN_DEACTIVATED_REALM_DELETION_DAYS = 14
+
+# Maximum number of days that can be given in realm data
+# deletion schedule when deactivating a realm.
+# Give None if there is no maximum value.
+# MAX_DEACTIVATED_REALM_DELETION_DAYS= None
```

Comment:
We put two `#` before comments, and we should make clear the _reason_ you would (or would not) set None or 0 for these, not just what they entail.

```suggestion
### Data retention settings
## Minimum number of days a realm administrator can
## choose to have data deleted after.  Set to 0 to
## allow realm administrators to deactivate and
## immediately irrevocably delete all data.
# MIN_DEACTIVATED_REALM_DELETION_DAYS = 14

## Maximum number of days a realm administrator can
## choose to have data deleted after.  Set to None
## to allow realm administrators to never delete data.
# MAX_DEACTIVATED_REALM_DELETION_DAYS= None